tools/df/docker/bird-fi-test/run.sh

```diff
@@ -0,0 +1,10 @@
+IMAGE=bird-fi-test
+CONTAINER=bird-fi-test
+
+set -e
+docker run --detach --name ${CONTAINER} ${IMAGE}
+docker exec ${CONTAINER} bash -c 'echo $PATH'
+docker exec ${CONTAINER} bash -c "/usr/local/sbin/bird -c /usr/local/etc/bird.conf"
+docker exec ${CONTAINER} bash -c "/usr/local/sbin/birdc show proto"
+BGP_IP=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' ${CONTAINER})
+docker exec ${CONTAINER} bash -c "env BGP_IP=$BGP_IP echo \"fi tests here and bgp on ${BGP_IP}\""
```

Comment:
Still not sure how to find a suitable environment for this to run in. Our current infrastructure has all FI test jenkins jobs running inside docker containers. I double-checked with @wuillaum -- can't run docker in docker. I can make a TI ticket for investigating running FI tests in an EC2 instance.

Reply:
run is the equivalent of the jenkins pipeline script - currently of course only to be run in your development machine.